### ISSUE TITLE
[Windows 10 pro] Application getting cutoff if taskbar is vertical

### ISSUE THREAD

**Issue by tboulis** [maxogden/menubar#298]
### Description
The menubar window is not full visible (bottom is getting cut off) with this arrangement:
- Windows 10 Pro
- 3 monitors (1 laptop, 2 standalone)
- Windows task bar is on the right side of the screen (vertical)

### Specifications
- Menubar version: `9.0.1`
- Platform: `Windows 10 Pro version 1083`
- Electron version:` 9.2.1`

### Other information
Could be related to this PR: https://github.com/maxogden/menubar/pull/217

### Screenshot
![image](https://user-images.githubusercontent.com/24255041/93453983-01d19b00-f8e3-11ea-9b3d-076bcf00bd60.png)

**Comment by tboulis** [comment]
@amaurymartiny fyi

**Comment by amaurymartiny** [comment]
Yes, I believe that code was written before I became maintainer. 

> Maybe would another way be more effective?

Feel free to improve with a PR!

**Comment by semenov-oleg** [comment]
Ok. I'm going to remove this code. I've checked locally on Windows with 2 monitors. Seems that project works fine without this 'optimization'
